docs/integrations/presto_trino.md

```diff
@@ -1,50 +1,54 @@
 ---
 layout: default
-title: Presto
-description: This section covers how you can start using lakeFS with Presto, is an open source, distributed SQL query engin.
+title: Presto/Trino
+description: This section covers how you can start using lakeFS with Presto, is an open source, distributed SQL query engine
 parent: Integrations
 nav_order: 40
 has_children: false
 redirect_from: ../using/presto.html
 ---
 
-# Using lakeFS with Presto
+# Using lakeFS with Presto/Trino
+
 {: .no_toc }
-[Presto](https://prestodb.io/) is a distributed SQL query engine designed to query large data sets distributed over one or more heterogeneous data sources.
+[Presto](https://prestodb.io){:target="_blank"} and [Trino](https://trinodb.io){:target="_blank"} are a distributed SQL query engine designed to query large data sets distributed over one or more heterogeneous data sources.
 {: .pb-5 }
 
 ## Table of contents
+
 {: .no_toc .text-delta }
 
 1. TOC
 {:toc .pb-5 }
 
-Querying data in lakeFS from Presto is the same as querying data in S3 from Presto.
-It is done using the [Hive connector](https://prestodb.io/docs/current/connector/hive.html).
+Querying data in lakeFS from Presto/Trino is the same as querying data in S3 from Presto/Trino. It is done using the [Presto Hive connector](https://prestodb.io/docs/current/connector/hive.html){:target="_blank"} or [Trino Hive connector](https://trino.io/docs/current/connector/hive.html){:target="_blank"}.
 
  **Note** 
  In the following examples we set AWS credentials at runtime, for clarity. In production, these properties should be set using one of Hadoop's standard ways of [Authenticating with S3](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html#Authenticating_with_S3){:target="_blank"}. 
  {: .note}
+
 ## Configuration
 
 ### Configure Hive connector
-Create ```/etc/catalog/hive.properties``` with the following contents to mount the ```hive-hadoop2``` connector as the ```hive``` catalog, replacing ```example.net:9083``` with the correct host and port for your Hive metastore Thrift service:
+
+Create `/etc/catalog/hive.properties` with the following contents to mount the `hive-hadoop2` connector as the `hive` catalog, replacing `example.net:9083` with the correct host and port for your Hive metastore Thrift service:
 ```properties
 connector.name=hive-hadoop2
 hive.metastore.uri=thrift://example.net:9083
 ```
 
```

Comment:
I think we should relate to Glue metastore here as well

Reply:
Both Trino and Presto documentation use the above instructions to set up the Hive connector as the first step.